src/test/java/io/appium/java_client/ChromeDriverPathUtil.java

```diff
@@ -0,0 +1,34 @@
+package io.appium.java_client;
+
+import static java.nio.file.FileSystems.getDefault;
+import static org.openqa.selenium.Platform.MAC;
+import static org.openqa.selenium.Platform.WINDOWS;
+import static org.openqa.selenium.Platform.getCurrent;
+
+import org.openqa.selenium.Platform;
+
+import java.io.File;
+import java.nio.file.Path;
+
+public final class ChromeDriverPathUtil {
+    private static final Path ROOT_TEST_PATH = getDefault().getPath("src")
+            .resolve("test").resolve("java").resolve("io").resolve("appium").resolve("java_client");
+
+    /**
+     * @return the choromedriver file which depends on platform.
+     */
+    public static File getChromeDriver() {
+        Path resultPath;
+        Platform current = getCurrent();
+
+        if (current.is(WINDOWS)) {
```

Comment:
why cannot we use `switch () case ...: return` here? this will allow to get rid of many temporary variables

Reply:
@mykola-mokhnach 

Because there may be WINDOWS8, WINDOWS10, WINDOWS_XP etc. I think that it is easier to compare current platforn with WINDOWS (the result for WINDOWS8, WINDOWS10, WINDOWS_XP is TRUE)

Reply:
np if you prefer this. However it still can be simplified by retuning the value immediately inside if block